Orange/data/dask.py

```diff
@@ -8,6 +9,33 @@
 import pandas
 
 from Orange.data import Table
+from Orange.widgets.utils.messages import UnboundMsg
```

Comment:
This code should not know (import) anything about the widget. Please move this code somewhere else.